pymc3/tests/test_distributions.py

```diff
@@ -941,23 +947,27 @@ def test_chi_squared(self):
             lambda value, nu: sp.chi2.logpdf(value, df=nu),
         )
 
-    @pytest.mark.xfail(
-        condition=(aesara.config.floatX == "float32"),
-        reason="Poor CDF in SciPy. See scipy/scipy#869 for details.",
-    )
-    def test_wald_scipy(self):
+    def test_wald_scipy_logp(self):
         self.check_logp(
             Wald,
             Rplus,
             {"mu": Rplus, "alpha": Rplus},
             lambda value, mu, alpha: sp.invgauss.logpdf(value, mu=mu, loc=alpha),
             decimal=select_by_precision(float64=6, float32=1),
+            n_samples=-1,
         )
+
+    @pytest.mark.xfail(
+        condition=(aesara.config.floatX == "float32"),
+        reason="Poor CDF in SciPy. See scipy/scipy#869 for details.",
+    )
+    def test_wald_scipy_logcdf(self):
         self.check_logcdf(
             Wald,
             Rplus,
             {"mu": Rplus, "alpha": Rplus},
             lambda value, mu, alpha: sp.invgauss.logcdf(value, mu=mu, loc=alpha),
+            n_samples=-1,
```

Comment:
Xfailed as expected in https://github.com/pymc-devs/pymc3/pull/4497/checks?check_run_id=2021551266